### DISCUSSION TITLE
whent I open extention manager crash at all times

### DISCUSSION THREAD

**Discussion by Cafesail** [pawelsalawa/sqlitestudio#3822] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
In previous bug report #3821 you claim that the app acrashes all the time, but here it seems you got it running, it just crashed when opening extension manager?

Did you add any extension there previously? Or did it crash even the first time you tried to open it?
